src/babel/util.js

```diff
@@ -90,13 +90,24 @@ export function regexify(val: any): RegExp {
   throw new TypeError("illegal type for regexify");
 }
 
-export function arrayify(val: any, mapFn?: Function): Array {
+export function arrayify(val: any, mapFn?: Function, mapExcludeFn?: Function): Array {
   if (!val) return [];
   if (isBoolean(val)) return arrayify([val], mapFn);
   if (isString(val)) return arrayify(list(val), mapFn);
 
   if (Array.isArray(val)) {
-    if (mapFn) val = val.map(mapFn);
+    if (mapExcludeFn && mapFn) {
+      val = val.map(function (v) {
+        if (mapExcludeFn(v)) {
+          return v;
+        } else {
+          mapFn.apply(null, arguments);
```

Comment:
It should. Sorry about the bad PR.
I think I should add more and better tests while I am at it. :open_mouth: